lib/git-version-bump.rb

```diff
@@ -224,6 +224,32 @@ def self.run_command(cmd, desc)
 	end
 	private_class_method :run_command
 
+        # Execute a command, specified as an array but retain ability to run the editor for notes.
+        #
+        # On success, control is returned.
+        # On error, a `CommandFailure` exception is raised.
+        #
+        def self.run_command_retain_tty(cmd, desc)
+                unless cmd.is_a?(Array)
+                        raise ArgumentError, "Must pass command line arguments in an array"
+                end
+
+                unless cmd.all? { |s| s.is_a?(String) }
+                        raise ArgumentError, "Command line arguments must be strings"
+                end
+
+                if debug?
+                        p :GVB_CMD, desc, cmd
+                end
+
+                pid = spawn(*cmd)
+                status = Process.wait pid
+
+                if status.exitstatus != 0
+                        raise CommandFailure.new("Failed while #{desc}", out, status.exitstatus)
```

Comment:
Since we're not capturing the output here, the `out` variable won't be initialized.
```suggestion
                        raise CommandFailure.new("Failed while #{desc}", "(output not captured)", status.exitstatus)
```